nominal/_utils.py

```diff
@@ -169,6 +175,7 @@ class FileTypes:
     CSV_GZ: FileType = FileType(".csv.gz", "text/csv")
     # https://issues.apache.org/jira/browse/PARQUET-1889
     PARQUET: FileType = FileType(".parquet", "application/vnd.apache.parquet")
+    MP4: FileType = FileType(".mp4", "video/mp4")
```

Comment:
Is it worth making MOV and/or AVI container types a first-class FileType?

Reply:
This enum is actually just for default/well-known values. When using the functions that actually take a path, it's parsed from the path.